### PR TITLE
Readme: Update readme with snap remove command

### PR DESCRIPTION
**Pull request by nitkon**
Update Readme with command on 
how to remove snap after installing it. 

Fixes: #446

Signed-off-by: Nitesh Konkar niteshkonkar@in.ibm.com

### CODE REVIEW

File: snap/README.md [hunk 1]
```diff
@@ -64,6 +65,14 @@ The path to the runtime provided by the Kata Containers snap image is
 `/snap/kata-containers/current/usr/bin/kata-runtime`. You should use it to
 run Kata Containers with [docker][9] and [kubernetes][10].
 
+## Remove snap
+
+You can remove Kata Containers by running the following command:
```
Comment (grahamwhaley) [inline snap/README.md]: s/Kata Containers/the Kata Containers snap/

Comment (nitkon) [reply snap/README.md]: done.

**Pull request merged**